src/test/java/com/infobip/interviewdemo/service/CommentServiceTest.java

```diff
@@ -0,0 +1,109 @@
+package com.infobip.interviewdemo.service;
+
+import com.infobip.interviewdemo.domain.CodeReviewEntity;
+import com.infobip.interviewdemo.dto.CommentRequest;
+import com.infobip.interviewdemo.repository.CodeReviewRepository;
+import com.infobip.interviewdemo.repository.CommentRepository;
+import org.assertj.core.api.WithAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.Instant;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class CommentServiceTest implements WithAssertions {
+
+    @Mock
+    private CommentRepository commentRepository;
+    @Mock
+    private CodeReviewRepository codeReviewRepository;
+
+    @Mock
+    private AnalyticsService analyticsService;
+
+    @InjectMocks
+    private CommentService underTest;
+
+    @BeforeEach
+    void init(){
+        var codeReviewEntity = new CodeReviewEntity();
```

Comment:
It's local var and it's not used. As it is only line in this method, you can delete whole method.